src/DropDownMenu/DropDownMenu.jsx

```diff
@@ -235,10 +189,6 @@ const DropDownMenu = React.createClass({
    */
   getInputNode() {
```

Comment:
@newoga, I tried to remove this method once. It bit me so hard I spent a day in hospital :laughing: This method is used by the `TextField` to get the underlying `input` so it can set or get it's value. your proposal for the TextField will solve a lot of issues like this. And I can say, this is a design flaw and it has manifested inself into a monstrosity :fearful: :fearful: